permissions/plugin-job-config-history.yml

```
@@ -0,0 +1,11 @@
+---
+name: "job-config-history"
+github: "jenkinsci/job-config-history-plugin"
+issues:
+- jira: '15683' # jobconfighistory-plugin
+paths:
+- "org/jenkins-ci/plugins/job-config-history"
```

you can't rename the artifact ID as there's no way to upgrade from that but we can rename the repo that's fine, although you may want to keep the artifact id in the file title, i.e. not rename this file.
```suggestion
- "org/jenkins-ci/plugins/jobConfigHistory"
```

---

Good call, reverted that back 👍🏻